docs/zh/use_case/JD_recommendation.md

```diff
@@ -1,158 +1,199 @@
+如何从历史数据中找出规律，去预测用户未来的购买需求，让最合适的商品遇见最需要的人，是大数据应用在精准营销中的关键问题，也是所有电商平台在做智能化升级时所需要的核心技术。京东作为中国最大的自营式电商，沉淀了数亿的忠实用户，积累了海量的真实数据。
 
-## 1.  环境准备
-### 1.1 下载demo演示用的数据与脚本
+本案例以京东商城真实的用户、商品和行为数据（脱敏后）为基础，通过数据挖掘技术和机器学习算法，构建用户购买商品的预测模型，输出高潜用户和目标商品的匹配结果，为精准营销提供高质量的目标群体，挖掘数据背后潜在的意义，为电商用户提供更简单、快捷、省心的购物体验。
 
-下载demo演示用的数据、脚本以及推理所需的OneEmbedding库（详情见[配置oneflow推理服务](#33-配置oneflow推理服务)），在后面的步骤中可以直接使用。
+本案例使用 OpenMLDB 进行数据挖掘，使用 OneFlow 中的 [DeepFM](https://github.com/OneFlow-Inc/models/tree/main/RecommenderSystems/deepfm) 模型进行高性能训练推理，提供精准的商品推荐。
+
+```{note}
+
+注意，本文档使用的是预编译好的 Docker 镜像。如果希望在自己编译和搭建的 OpenMLDB 环境下进行测试，需要配置使用[面向特征工程优化的 Spark 发行版](https://openmldb.ai/docs/zh/main/tutorial/openmldbspark_distribution.html)。请参考[针对 OpenMLDB 优化的 Spark 发行版文档](https://openmldb.ai/docs/zh/main/deploy/compile.html#openmldb-spark)和[安装部署文档](https://openmldb.ai/docs/zh/main/deploy/install_deploy.html)。
+
+```
+
+## 准备
+
+### 下载数据与脚本
+
+下载数据、脚本以及推理所需的 OneEmbedding 库（详情见[配置 OneFlow 推理服务](#配置-OneFlow-推理服务)），在后面的步骤中可以直接使用。
 
 ```
 wget http://openmldb.ai/download/jd-recommendation/demo.tgz
 tar xzf demo.tgz
 ls demo
 ```
 
-也可以checkout Github仓库中的`demo/jd-recommendation`。
-我们将这个`demo`目录定为环境变量`demodir`，之后的脚本中多会使用这一环境变量。所以，你需要配置这一变量：
+也可以 checkout GitHub 仓库中的 `demo/jd-recommendation`。
+这个 `demo` 目录定为环境变量 `demodir`，之后的脚本中多会使用这一环境变量。所以，你需要配置这一变量：
+
 ```
 export demodir=<your_path>/demo
 ```
 
-我们仅使用小数据集做演示。如果你想要使用全量数据集，请下载[JD_data](http://openmldb.ai/download/jd-recommendation/JD_data.tgz)。
+本例仅使用小数据集做演示。如果你想要使用全量数据集，请下载 [JD_data](http://openmldb.ai/download/jd-recommendation/JD_data.tgz)。
 
-### 1.2 OneFlow工具包安装
+### 安装 OneFlow 工具包
 
-OneFlow工具依赖GPU的强大算力，所以请确保部署机器具备Nvidia GPU，并且保证驱动版本 >=460.X.X  [驱动版本需支持CUDA 11.0](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-major-component-versions)。
+OneFlow 工具依赖 GPU 的强大算力，所以请确保部署机器具备 Nvidia GPU，并且保证驱动版本 >=460.X.X。[驱动版本需支持 CUDA 11.0](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-major-component-versions)。
 
-我们推荐使用conda来管理oneflow环境，安装OneFlow开发版（支持oneembedding），以及本案例演示所需的其他依赖，手动创建方法如下：
+推荐使用 conda 来管理 OneFlow 环境，安装 OneFlow 开发版（支持 OneEmbedding），以及本案例演示所需的其他依赖，手动创建方法如下：
 
 ```bash
 conda create -y -n oneflow python=3.9.2
 conda activate oneflow
-pip install -f https://staging.oneflow.info/branch/master/cu112 --pre oneflow
+pip install -f https://staging.oneflow.info/branch/master/cu112 --pre OneFlow
 pip install psutil petastorm pandas sklearn xxhash "tritonclient[all]" geventhttpclient tornado
 ```
 
-拉取Oneflow-serving镜像：
+拉取 oneflow_serving 镜像：
+
 ```bash
-docker pull oneflowinc/oneflow-serving:nightly
+docker pull oneflowinc/oneflow_serving:nightly
```

Comment:
pls double check, in general you should not modify anything in the command, unless the content is wrong